xml/System/Char.xml

```diff
@@ -4945,9 +4879,9 @@ This member is an explicit interface member implementation. It can be used only
 ## Examples
  The following code example demonstrates <xref:System.Char.ToString%2A>.
 
- [!code-cpp[System.Char.ToString#17](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Char.ToString/CPP/tostring.cpp#17)]
- [!code-csharp[System.Char.ToString#17](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.ToString/CS/tostring.cs#17)]
- [!code-vb[System.Char.ToString#17](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Char.ToString/VB/tostring.vb#17)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Char.ToString/CPP/tostring.cpp" id="Snippet17":::
```

Comment:
This one was not added to the MemberGroup element

Reply:
done